apps/studio/src/examples/websocket-gemini.yml

```diff
@@ -94,25 +148,29 @@ channels:
               type: boolean
               default: true
               description: Include auction events
-    subscribe:
-      summary: Receive market updates on a given symbol
-      message:
-        $ref: '#/components/messages/marketData'
-
-#
-# All reusable parts for readability and staying DRY
-#
+operations:
+  sendMarketData:
+    action: send
+    channel:
+      $ref: '#/channels/marketDataV1'
+    summary: Receive market updates on a given symbol
+    messages:
+      - $ref: '#/components/messages/marketData'
 components:
   messages:
     marketData:
       summary: Message with marked data information.
-      description: |
-        The initial response message will show the existing state of the order book. Subsequent messages will show all executed trades, as well as all other changes to the order book from orders placed or canceled.
+      description: >
+        The initial response message will show the existing state of the order
+        book. Subsequent messages will show all executed trades, as well as all
+        other changes to the order book from orders placed or canceled.
       payload:
         $ref: '#/components/schemas/market'
       examples:
         - name: updateMessage
-          summary: Example of an update message that contains a change in price information.
+          summary: >-
+            Example of an update message that contains a change in price
+            information.
           payload:
             type: update
             eventId: 36902233362
```

Comment:
`timestamp` and `timestampms` should be string in `date-time` and `time` format accordingly, according the schema mentioned.